model/system/metrics.yaml

```diff
@@ -29,6 +31,7 @@ groups:
   - id: metric.system.cpu.logical.count
     type: metric
     metric_name: system.cpu.logical.count
+    value_type: int
```

Comment:
/cc @open-telemetry/semconv-system-approvers is this right?

Reply:
To my knowledge this is correct. Semi-related to @trask's comment below, but none of the raw system APIs we would use to instrument this metric will produce anything but an integer count of CPU cores. This is different for container environments like k8s though (and I see they are correctly marked as `double` within this PR).

Reply:
thanks!

resolving this since we're not going to require types to match, it's just a hint for codegen and can be ignored. - https://github.com/open-telemetry/semantic-conventions/pull/2444#issuecomment-3029374240